modules/rostests/apitests/ntdll/RtlGenerate8dot3Name.c

```diff
@@ -70,13 +70,13 @@ PWSTR LongShortNames[LONG_NAMES_COUNT] =
 
 PVOID LoadCodePageData(ULONG Code)
 {
-    char filename[MAX_PATH], sysdir[MAX_PATH];
+    char filename[MAX_PATH + 17], sysdir[MAX_PATH];
```

Comment:
For what it's worth, this warning is stupid, and thus, the "fix" makes no sense.
I'd rather leave the filename[MAX_PATH] and change the call to GetSystemDirectory(sysdir, MAX_PATH - 17).